src/components/layer-controller/LayerController.js

```diff
@@ -16,123 +16,114 @@ import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import ChannelController from './ChannelController';
 import LayerOptions from './LayerOptions';
 
-import { LAYER_ADD, LAYER_CHANGE, METADATA_ADD } from '../../events';
+import { LAYER_CHANGE } from '../../events';
 import reducer from './reducer';
 import { useExpansionPanelStyles } from './styles';
+import {
+  GLOBAL_SLIDER_DIMENSION_FIELDS, DTYPE_VALUES, MAX_CHANNELS, DEFAULT_LAYER_PROPS,
+} from './constants';
 
-// For now these are the global channel selectors.
-// We can expand this part of the application as new needs arise.
-const GLOBAL_SLIDER_DIMENSION_FIELDS = ['z', 'time'];
-
-function buildDefaultSelection(imageDims, defaultIndex = 0) {
+// Return the midpoint of the global dimensions.
+function getDefaultGlobalSelection(imageDims) {
+  const globalIndices = imageDims.filter(dim => GLOBAL_SLIDER_DIMENSION_FIELDS.includes(dim.field));
   const selection = {};
-  imageDims.forEach((dim) => {
-    selection[dim.field] = defaultIndex;
+  globalIndices.forEach((dim) => {
+    selection[dim.field] = Math.floor((dim.values.length || 0) / 2);
```

Comment:
Default selection is just the middle? Why not `0`?

Reply:
OMERO defaults to the middle for z stack and starts at 0 for time.  Beyond that, just playing around with their/our data, the z-stack is often not very interesting (especially for confocal) in the first images, and you usually don't lose much by having time be the midpoint.

Reply:
So, the short answer is that it produces the nicest initial image from my experience.